src/netex_france/stops.rs

```diff
@@ -435,3 +435,86 @@ impl<'a> StopExporter<'a> {
             .build()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    mod valid_impaired_access {
+        use super::*;
+        use crate::{
+            minidom_utils::ElementWriter,
+            model::{Collections, Model},
+        };
+        use pretty_assertions::assert_eq;
+        use Availability::*;
+
+        fn get_mobility_impaired_access(element: Element) -> String {
+            let writer = ElementWriter::new(element, false);
+            let mut buffer = Vec::<u8>::new();
+            writer.write(&mut buffer).unwrap();
+            String::from_utf8(buffer)
+                .unwrap()
+                .replace(
+                    r#"<?xml version="1.0" encoding="UTF-8"?><MobilityImpairedAccess>"#,
+                    "",
+                )
+                .replace(r#"</MobilityImpairedAccess>"#, "")
+                .to_owned()
```

Comment:
Why do you serialize the `Element` here?  You can replace this whole function by the following code

```rust
element
    .nodes()
    .next()
    .unwrap()
    .as_text()
    .unwrap()
    .to_string()
```

Reply:
Commit updated https://github.com/CanalTP/transit_model/pull/563/commits/23653f76d31db8dfa35e28a908b0938ca9eaebf6